core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/visualization/htmlviz/HtmlVizOpExec.scala

```diff
@@ -0,0 +1,34 @@
+package edu.uci.ics.texera.workflow.operators.visualization.htmlviz
+
+import edu.uci.ics.amber.engine.common.InputExhausted
+import edu.uci.ics.amber.engine.common.virtualidentity.LinkIdentity
+import edu.uci.ics.texera.workflow.common.operators.OperatorExecutor
+import edu.uci.ics.texera.workflow.common.tuple.Tuple
+import edu.uci.ics.texera.workflow.common.tuple.schema.{Attribute, AttributeType}
+
+import scala.collection.Iterator
+import scala.util.Either
+
+/**
+  * HTML Visualization operator to render any given HTML code
+  */
+class HtmlVizOpExec(htmlContentColumnName: String) extends OperatorExecutor {
+
+  override def open(): Unit = {}
+
+  override def close(): Unit = {}
+
+  override def processTexeraTuple(
+      tuple: Either[Tuple, InputExhausted],
+      input: LinkIdentity
+  ): Iterator[Tuple] =
+    tuple match {
+      case Left(t) =>
+        val result = Tuple
+          .newBuilder()
+          .add("HTML_content", AttributeType.STRING, t.getField(htmlContentColumnName))
```

Comment:
please rename this to `HTML-Content`. We are trying to standardize column name format to use hyphens.

Reply:
Also, I am curious about what will happen if there are multiple tuples of html content? Does our frontend handle it? It's ok to assume that the html-operator expects just one tuple. If that is the case, please mention it using comments.

Reply:
you can pass multiple tuples, the frontend however as of now will only take in the first tuple